doc/api/events.markdown

```diff
@@ -295,6 +295,24 @@ they were registered, passing the supplied arguments to each.
 
 Returns `true` if event had listeners, `false` otherwise.
 
+### emitter.eventNames()
```

Comment:
@ChALkeR I am really sorry. I straight away started with the code. It makes sense. Thanks for explaining.